From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect linux/of.h include in tpm-chip
Patch-mainline: never, kabi
References: kabi

In 4.4.137, commit c7d581820c86b2b2e8c6eaa38bddb76273a2ca28 (tpm: do not
suspend/resume if power stays on), upstream commit
b5d0ebc99bf5d0801a5ecbe958caa3d68b8eaee8 added linux/of.h include to
tpm-chip.c and it made some of the symbols defined.

Protect the include by __GENKSYMS__ to satisfy the kABI checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/char/tpm/tpm-chip.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/char/tpm/tpm-chip.c
+++ b/drivers/char/tpm/tpm-chip.c
@@ -26,7 +26,9 @@
 #include <linux/spinlock.h>
 #include <linux/freezer.h>
 #include <linux/major.h>
+#ifndef __GENKSYMS__
 #include <linux/of.h>
+#endif
 #include "tpm.h"
 #include "tpm_eventlog.h"
 
